packages/fast-usdc/src/exos/settler.js

```diff
@@ -151,17 +151,23 @@ export const prepareSettler = (
             return;
           }
 
-          if (!addressTools.hasQueryParams(tx.receiver)) {
+          let endUserDestination;
+          try {
+            const { EUD } = decodeAddressHook(tx.receiver).query;
+            if (!EUD) {
+              console.log('no EUD parameter', tx.receiver);
```

Comment:
we want the tracer here, right?

```suggestion
              log('no EUD parameter', tx.receiver);
```

Reply:
This was copied over but yes, updated